testinfra/conftest.py

```diff
@@ -26,19 +25,17 @@ def securedrop_import_testinfra_vars(hostname, with_header=False):
     Vars must be stored in `testinfra/vars/<hostname>.yml`.
     """
     filepath = os.path.join(os.path.dirname(__file__), "vars", hostname+".yml")
-    file_load = open(filepath, 'r')
+    hostvars = yaml.load(open(filepath, 'r'))
```

Comment:
Prefer `yaml.safe_load`, unless you really need `load`: https://security.openstack.org/guidelines/dg_avoid-dangerous-input-parsing-libraries.html.